hass_nabucasa/auth.py

```diff
@@ -94,12 +94,24 @@ async def on_disconnect(self):
         """When the instance is disconnected."""
         self._refresh_task.cancel()
 
-    async def async_register(self, email, password):
+    async def async_register(
+        self,
+        email,
+        password,
+        attr_map=None,
```

Comment:
We don't need that. I'm a bit scared that it get misunderstood later and someone tries to use that without knowing exactly what it does in the library. I would vote to remote that.

Reply:
https://github.com/NabuCasa/hass-nabucasa/pull/306